packages/desktop-gui/src/prompts/prompts.jsx

```diff
@@ -0,0 +1,362 @@
+      link: {
+        url: 'https://on.cypress.io/bitbucket-pipelines',
+        params: {
+          utm_medium: this.utm_medium,
+          utm_campaign: 'Bitbucket',
+        },
+      },
+    },
+    {
+      name: 'GitLab CI/CD',
+      icon: GitLab,
+      link: {
+        url: 'https://on.cypress.io/gitlab-ci',
+        params: {
+          utm_medium: this.utm_medium,
+          utm_campaign: 'GitLab',
+        },
+      },
+    },
+    {
+      name: 'AWS CodeBuild',
+      icon: AWSCodeBuild,
+      link: {
+        url: 'https://on.cypress.io/aws-codebuild',
+        params: {
+          utm_medium: this.utm_medium,
+          utm_campaign: 'AWS',
+        },
+      },
+    },
+  ]
+
+  _close = () => {
+    this.props.prompts.closePrompt(this.slug)
+    ipc.setPromptShown(this.slug)
+  }
+
+  _openProviderLink = (link) => {
+    ipc.externalOpen(link)
+  }
+
+  _seeOther = () => {
+    ipc.externalOpen({
+      url: 'https://on.cypress.io/setup-ci',
+      params: {
+        utm_medium: this.utm_medium,
+        utm_campaign: 'Other',
+      },
+    })
+  }
+
+  _viewMore = () => {
+    ipc.externalOpen({
+      url: 'https://on.cypress.io/ci',
+      params: {
+        utm_medium: this.utm_medium,
+        utm_campaign: 'Learn More',
+      },
+    })
+  }
+
+  render () {
+    const { prompts, referenceElement } = this.props
+
+    return (
+      <Prompt
+        isOpen={prompts[this.slug]}
+        referenceElement={referenceElement}
+        className='prompt-ci1'
+      >
+        <div className='prompt-body'>
+          <button className='btn btn-link close' onClick={this._close}>
+            <i className='fas fa-times' />
+          </button>
+          <div className='text-content'>
+            <h2>Setting up CI</h2>
+            <p>We've created these guides to help you optimize how you're running tests in CI.</p>
+          </div>
+          <div className='ci-providers'>
+            { this.ciProviders.map((provider) => {
+              const { name, icon: Icon, link } = provider
+
+              return (
+                <button className='ci-provider-button' onClick={() => this._openProviderLink(link)} key={name}>
+                  <Icon width={18} height={18} />
+                  <span>{name}</span>
+                </button>
+              )
+            }) }
+            <button className='btn btn-link see-other-guides' onClick={this._seeOther}>
+              <span>See other guides</span> <i className='fas fa-arrow-right' />
+            </button>
+          </div>
+          <div className='prompt-buttons'>
+            <button className='btn btn-success' onClick={this._viewMore}>Learn More</button>
+            <br />
+            <button className='btn btn-link' onClick={this._close}>
+              Close
+            </button>
+          </div>
+        </div>
+      </Prompt>
+    )
+  }
+}
+
+@observer
+// eslint-disable-next-line no-unused-vars
+class DashboardPrompt1 extends Component {
```

Comment:
I would prefer to remove code if it's not being used. Or comment out since this is already a larger file.